dist/angular-content-editable.js

```diff
@@ -145,7 +145,7 @@ angular.module('angular-content-editable')
             // update elem html value
             elem.html(html);
```

Comment:
[just a thought] I wonder if we should change this to use `elem.text(text)` and remove all the strip-replace logic since we don't really want to use this to edit real html anyway

Reply:
I guess we could but then I'd want to do some more thorough testing as well as strongly consider writing our own directive (as we've removed most logic ;)) - for now I think I made the smallest cut that gets us what we need - once we move on to React all will be good ;)